### ISSUE TITLE
more-info attachment in text/plain ...decoded from Base64?

### ISSUE THREAD

**Issue by osher** [gkushang/cucumber-html-reporter#72]
## In short
The 'more info' section is epected to show text, but it's encoding is disrupted.

I expect:
```
some text!!!!!!\nand more text!!",
```
But I get:
```
�����mjwf���{
```

I suspect the Base64 decoding in here:
https://github.com/gkushang/cucumber-html-reporter/blob/develop/lib/reporter.js#L147-L150
But I could just be wrong.

Can anybody have a look?

## Here's the full story:

Given this step def:
```
require('cucumber')
.defineSupportCode(function({defineStep}) {
  defineStep(/bar/, function(t) {
      var attach = this.attach;
      console.log("bar", arguments)
      this.attach("some text!!!!!!\nand more text!!")
      this.attach("some text!!!!!!\nand more text!!", "text/plain")
      this.attach(require('fs').createReadStream("./tmp.jpg"), "image/jpg")
  })
})
```
Given  *tmp.jpg* is found on disk, it's just some small icon.

Given this scenario:
```
Feature: foo
  Scenario: bar
    Given bar "aha"
```
Given the cuc is run with the following command:
```
cucumberjs --format json:tmp.json --format pretty && node bin/gen-html.js
```
And the report generation code in `bin/gen-html.js` is:
```
require('cucumber-html-reporter')
.generate({
  theme:          "bootstrap",
  storeScreenShots: true,
  jsonFile:       "./tmp.json",
  output:         "./cuc.html",
  launchReport:   false,
  reportSuiteAsScenarios:
                  true
});
```

Then - the "+more info" box should show 
```
some text!!!!!!
and more text!!
some text!!!!!!
and more text!!
```

Instead I get:
```
�����mjwf���{
�����mjwf���{
```

I can see the embedding section in the json report:
```
            "embeddings": [
              {
                "data": "some text!!!!!!\nand more text!!",
                "mime_type": "text/plain"
              },
              {
                "data": "some text!!!!!!\nand more text!!",
                "mime_type": "text/plain"
              },
              {
                "data": "/9j/4AAQSkZJRgABAQAAA.....
                "mime_type": "image/jpg"
              }
            ]
```

**Comment by gkushang** [comment]
https://github.com/gkushang/cucumber-html-reporter/pull/81 will fix this issue

**Comment by gkushang** [comment]
published the fix with `cucumber-html-reporter@2.0`. Please try it out, and let us know how it works.
